web/package.json

```diff
@@ -11,20 +11,23 @@
     "url": "https://github.com/electricityMaps/electricitymaps-contrib.git"
   },
   "scripts": {
-    "mockserver": "nodemon ../mockserver/server.js",
```

Comment:
Turns out you were right about this one Mads 😅

I don't know why it worked when I originally tested it locally, but it works now.